src/ontology/efo-edit.owl

```diff
@@ -202619,6 +202626,21 @@ AnnotationAssertion(rdfs:label efo:EFO_0010971 "response to metamizole"@en)
 SubClassOf(efo:EFO_0010971 obo:GO_0042493)
 SubClassOf(efo:EFO_0010971 ObjectSomeValuesFrom(obo:IAO_0000136 obo:CHEBI_62088))
 
+# Class: efo:EFO_0010972 (blood phosphate measurement)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "CMO:0000504") obo:IAO_0000115 efo:EFO_0010972 "The amount of phosphorus, measured as inorganic phosphate, in a specified sample of blood.")
+AnnotationAssertion(obo:IAO_0000117 efo:EFO_0010972 "Sam Lambert")
+AnnotationAssertion(efo:gwas_trait efo:EFO_0010972 "true"^^xsd:boolean)
+AnnotationAssertion(oboInOwl:hasDbXref efo:EFO_0010972 "ORCID:0000-0001-8222-008X")
+AnnotationAssertion(oboInOwl:hasDbXref efo:EFO_0010972 "PMID:33462484")
+AnnotationAssertion(Annotation(rdfs:comment "CMO:0000542") oboInOwl:hasExactSynonym efo:EFO_0010972 "Serum phosphate level")
+AnnotationAssertion(Annotation(rdfs:comment "CMO:0000504") oboInOwl:hasExactSynonym efo:EFO_0010972 "blood phosphate level")
+AnnotationAssertion(Annotation(rdfs:comment "https://www.ncbi.nlm.nih.gov/books/NBK310/") oboInOwl:hasExactSynonym efo:EFO_0010972 "serum inorganic phosphate concentration")
+AnnotationAssertion(Annotation(rdfs:comment "https://www.ncbi.nlm.nih.gov/books/NBK310/") oboInOwl:hasExactSynonym efo:EFO_0010972 "serum inorganic phosphorus concentration")
```

Comment:
All rdfs:comment here should be database_cross_reference

Reply:
Thanks all replaced by DbXref and commited by pushing this branch
AnnotationAssertion(Annotation(oboInOwl:hasDbXref "CMO:0000542") oboInOwl:hasExactSynonym efo:EFO_0010972 "Serum phosphate level")
AnnotationAssertion(Annotation(oboInOwl:hasDbXref "CMO:0000504") oboInOwl:hasExactSynonym efo:EFO_0010972 "blood phosphate level")
AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://www.ncbi.nlm.nih.gov/books/NBK310/") oboInOwl:hasExactSynonym efo:EFO_0010972 "serum inorganic phosphate concentration")
AnnotationAssertion(Annotation(oboInOwl:hasDbXref "https://www.ncbi.nlm.nih.gov/books/NBK310/") oboInOwl:hasExactSynonym efo:EFO_0010972 "serum inorganic phosphorus concentration")